### PR TITLE
Random numbers data provider

### PR DESCRIPTION
**Pull request by zlumer**
Added random numbers data provider (not parametrized at the moment)

### CODE REVIEW

File: src/api/configurator.ts [hunk 1]
```diff
@@ -61,7 +61,12 @@ export async function makeConfig(): Promise<IConfigFunction>
 				name: "sports"
 			},
 			{
-				name: "random"
+				name: "random",
+				types: (() => { let arr = []; for (let i = 0; i < 10000; i++) { arr[i] = `${i}` } return arr })(),
```
Comment (Sindbag) [inline src/api/configurator.ts]: Hope, this is temporary solution :)

File: src/app.ts [hunk 2]
```diff
@@ -8,6 +8,7 @@ import { getDataDefByHash } from "./reverse_map"
 import { providers } from "./providers"
 import { app as api, CONFIG as apiConfig } from "./api"
 import { generate, makeConfig } from "./api/configurator"
+import { IDataProvider } from "./IDataProvider"
 
 console.log("hello")
```
Comment (Sindbag) [inline src/app.ts]: is it really needed here?

File: src/api/configurator.ts (hunk 1 above)
Comment (zlumer) [reply src/api/configurator.ts]: Definitely. We're going to add parametrized data requests soon.

File: src/app.ts [hunk 3]
```diff
@@ -20,14 +21,16 @@ let writers = {
 	eth: ethPush
 }
 
-let onRequest: RequestHandler = async req => {
+export let onRequest: RequestHandler = async req => {
```
Comment (Sindbag) [inline src/app.ts]: may be make it `async ({receiver, blockchain, dataHash})` ?

File: src/reverse_map.ts [hunk 4]
```diff
@@ -10,13 +11,26 @@ export async function init()
 		.map(x => x.join('/'))
 		.map(x => ({ category: "crypto", provider: "binance", ident: x, type: "price" as IDataType}))
 		.map(x => ({ ...x, hash: hashDataId(x) }))
-		.reduce((acc, cur) => ({ ...acc, [cur.hash]: cur }), {} as { [key: string]: IDataDefinition })
+		.reduce((acc, cur) => ({ ...acc, [cur.hash]: cur }), dataDefinitions)
 	
 	dataDefinitions["363e7fe8b47534460fd06dafd5e18a542fe1aaa78038d7ca5e84694f99a788e5"] = dataDefinitions[hashDataId({
```
Comment (Sindbag) [inline src/reverse_map.ts]: hardcoded hash?

Comment (zlumer) [reply src/reverse_map.ts]: Data migration. Temporary workaround until we fully move to DB hash storage.

File: src/app.ts (hunk 2 above)
Comment (zlumer) [reply src/app.ts]: This log line indicates that the app started launching. Maybe wording (`hello`) may be better, e.g. `starting app`. This PR doesn't make any changes to this behavior, let's discuss it in another issue.

**Pull request merged**